.github/ISSUE_TEMPLATE/BUG_REPORT.yaml

```
@@ -0,0 +1,40 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug", "triage"]
+projects: ["symfony/ux"]
+body:
+  - type: markdown
+    attributes:
+    value: |
+      Thanks for taking the time to fill out this bug report!
+    - type: dropdown
+    id: package
+    attributes:
+      label: Package
+      description: What package is concerned by this $$
+      options:
+        - Autocomplete
+        - Live Component
+        - Icons
+        - Translator
+        - Twig Component
+        - Autre
+      default: 0
+    validations:
+      required: true
+    - type: checkboxes
+    id: terms
+    attributes:
+      label: Code of Conduct
+      description: By submitting this issue, you agree to follow our [Code of Conduct](https://example.com)
+      options:
+        - label: I agree to follow this project's Code of Conduct
+          required: true
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant log output
```

```suggestion
      label: Relevant log/stacktrace output
```

wdyt? a log can bo top level, a stack trace can help dig faster :)